tests/fixtures.py

```diff
@@ -680,7 +699,7 @@ def parametrize(params, values, **kwargs):
     return pytest.mark.parametrize(params, values, ids=ids, **kwargs)
 
 
-DYNAMIC_FEATURE_PRESENCE_TESTS = sorted(
+DYNAMIC_CAPE_FEATURE_PRESENCE_TESTS = sorted(
     [
         # file/string
         ("0000a657", "file", capa.features.common.String("T_Ba?.BcRJa"), True),
```

Comment:
could we reuse these for drakvuf or do results differ a lot?

Reply:
Hmm I assume I would need to reanalyze the sample using Drakvuf Sandbox, or analyze a different sample using both sandboxes and update this field. I'll try to work on this but I don't think it should be a blocker for now (can be added in another PR perhaps).

Reply:
please create an issue for this for tracking